docs/github_actions.rst

```diff
@@ -945,3 +945,30 @@ Generate code coverage reports once all tests have reported.
 Upload Coverage
 ---------------
 Upload the generated code coverage to `CodeCov <https://codecov.io/gh/QMCPACK/qmcpack/tree/develop/src>`_ where the badges on our repo will then be updated. Only done by jobs with name `*-coverage`.
+
+
+====== ====== ======== 
+Static Checks Workflow
+====== ====== ========
+
+A manually triggered workflow on the GitHub Actions tab can generate the required checks using the [clang-tidy](https://clang.llvm.org/extra/clang-tidy/) static analyzer. The current approach is to set checks in the `qmcpack/.clang-tidy` configuration file and run using `clang-tidy` v14 on GitHub Actions runners. The workflow is not part of CI, and it's currently used for reporting potential warnings on the GitHub Actions logs as they are addressed on the `develop` branch as part of refactoring efforts for code quality. 
+
+To run the workflow:
+- Go to the Actions tab
+- Click on the `static` workflow on the left
+- Click on `Run workflow` on the right
+- Use workflow from `Branch:develop` and click on the `Run workflow` button
+
+**Note:** the current `.clang-tidy` configuration file is compatible with clang v14 and runs on the `williamfgc/qmcpack-ci:ubuntu22-openmpi` docker image. To run locally on a Linux system use: `docker run -it user williamfgc/qmcpack-ci:ubuntu22-openmpi /bin/bash` or refer to the :ref:`running_docker`` section.
+
+To build locally enaling `clang-tidy`` static checks defined in `qmcpack/.clang-tidy` use the CMake `-DCMAKE_CXX_CLANG_TIDY` option as follows:
```

Comment:
`src/.clang-tidy` instead of `qmcpack/.clang-tidy`

Reply:
@ye-luo good point. @PDoakORNL @quantumsteve I'm unifying the `.clang-tidy` configuration. Are you OK removing the one under `/src`? The one at the top level would have similar checks.

Reply:
We should have only one configure file. I don't get why it should be under the root directory. If I understand correctly, clang-tidy handles C++ source code and all C++ code managed by QMCPACK is under src. Codes under external should not be subject to clang-tidy checks by QMCPACK.